homework-submission/week1-react/src/component/todos.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
In my opinion changing the path to be equivalent to the data it's inside well make a better understanding for the file structuring.